### PR TITLE
Add bindings for auto-package-update

### PR DESCRIPTION
**Pull request by condy0919**
@Ambrevar

### CODE REVIEW

**Comment by Ambrevar** [comment]
Is there no binding other than quit?

**Comment by condy0919** [comment]
Yes, only a `quit-window`. `auto-package-update.el` provides automatic package update without other user operations.

**Comment by Ambrevar** [comment]
Is it still worth a new file then?
@jojojames what do you think?

**Comment by condy0919** [comment]
Binding `q` to `quit-window` makes the behavior consistent.
If there is no such binding, user must use `evil-delete-window` (<kbd>C-w c</kbd>) or `delete-window` (<kbd>C-x 0</kbd>) which are inconvenient than pressing `q`.

File: modes/auto-package-update/evil-collection-auto-package-update.el [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+;; Evil bindings for auto-package-update.
+
+;;; Code:
+(require 'evil-collection)
+(require 'auto-package-update nil t)
+
+;;;###autoload
+(defun evil-collection-auto-package-update-setup ()
+  "Set up `evil' bindings for `auto-package-update'."
+  (evil-define-minor-mode-key 'normal 'auto-package-update-minor-mode
```
Comment (jojojames) [inline modes/auto-package-update/evil-collection-auto-package-update.el:36]: Can you route through to evil-collection-define-key instead?

I think you'll also want a defvar of the keymap.

See https://github.com/emacs-evil/evil-collection/blob/master/modes/hg-histedit/evil-collection-hg-histedit.el

**Comment by jojojames** [comment]
@Ambrevar Yeah, I think it's reasonable to add.

File: modes/auto-package-update/evil-collection-auto-package-update.el (hunk 1 above)
Comment (condy0919) [reply modes/auto-package-update/evil-collection-auto-package-update.el]: Finally done after found https://github.com/emacs-evil/evil/issues/301

**Comment by condy0919** [comment]
@Ambrevar

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**